install/v1alpha1-0.7.0/cstor-pool/pool.yaml

```diff
@@ -0,0 +1,399 @@
+---
+apiVersion: openebs.io/v1alpha1
+kind: CASTemplate
+metadata:
+  name: cstor-pool-create-default-0.7.0
+spec:
+  defaultConfig:
+  # CstorPoolImage is the container image that executes zpool replication and 
+  # communicates with cstor iscsi target
+  - name: CstorPoolImage
+    value: openebs/cstor-pool:ci
+  # CstorPoolMgmtImage runs cstor pool and cstor volume replica related CRUD 
+  # operations
+  - name: CstorPoolMgmtImage
+    value: openebs/cstor-pool-mgmt:cstor-feature-integration-ci
+  # HostPathType is a hostPath volume i.e. mounts a file or directory from the 
+  # host node’s filesystem into a Pod. 'DirectoryOrCreate' value  ensures 
+  # nothing exists at the given path i.e. an empty directory will be created.
+  - name: HostPathType
+    value: DirectoryOrCreate
+  # TODO
+  # This needs to be fetched from StorageClass
+  #
+  # RunNamespace is the namespace where namespaced resources related to pool 
+  # will be placed
+  - name: RunNamespace
+    value: openebs
+  taskNamespace: openebs
+  run:
+    tasks:
+    # Following are the list of run tasks executed in this order to 
+    # create a cstor storage pool
+    - cstor-pool-create-listdisk-default-0.7.0
+    - cstor-pool-create-listnode-default-0.7.0
+    - cstor-pool-create-putcstorpoolcr-default-0.7.0
+    - cstor-pool-create-putcstorpooldeployment-default-0.7.0
+    - cstor-pool-create-putstoragepoolcr-default-0.7.0
+    - cstor-pool-create-patchstoragepoolclaim-default-0.7.0
+---
+apiVersion: openebs.io/v1alpha1
+kind: CASTemplate
+metadata:
+  name: cstor-pool-delete-default-0.7.0
+spec:
+  defaultConfig:
+    # TODO
+    # This needs to be fetched from StorageClass
```

Comment:
Again , all the pool related information will come from spc only.
@AmitKumarDas 
@avishnu What is your stand here ?